controller/tests/models/domain/rsptag_test.py

```diff
@@ -20,16 +20,12 @@ def test_tag_ordering() -> None:
     """Test comparisons of RSPImageTag objects."""
     one = RSPImageTag.from_str("r21_0_1")
     two = RSPImageTag.from_str("r21_0_2")
-    assert one == one
-    assert one <= one
-    assert one >= one
```

Comment:
These deletions are incorrect. Those tests should stay. They're testing the implementation of the `__eq__` method on the abstract data type.